src/PHPUnit/Listeners/NaughtyTestListener.php

```diff
@@ -119,6 +123,11 @@ public function endTestSuite(TestSuite $suite)
                 $this->printNaughtyTest('Global TestSuite', $modifications);
             }
         }
+
+        if ($this->testSuiteCounter === 0 && $this->messages) {
```

Comment:
i think i prefer to have strictly boolean operands in the if conditions, e.g. `count($this->messages) > 0`. what do you think?